models/models_methods.go

```diff
@@ -288,6 +288,15 @@ func (workflow *WorkflowModel) Validate() ([]string, error) {
 			return warnings, err
 		}
 
+		stepIDData, err := CreateStepIDDataFromString(stepID, defaultStepLibSource)
+		if err != nil {
+			return warnings, err
+		}
+
+		if err := stepIDData.validate(defaultStepLibSource); err != nil {
+			return warnings, fmt.Errorf("invalid version format (%s) specified for step ID: %s", stepIDData.Version, stepIDData.IDorURI)
```

Comment:
it is a generic `stepIDData` validator, we can not be sure that only the version is wrong